pulpcore/app/viewsets/publication.py

```diff
@@ -70,12 +78,70 @@ class Meta:
 
 
 class DistributionViewSet(NamedModelViewSet,
-                          mixins.CreateModelMixin,
                           mixins.UpdateModelMixin,
                           mixins.RetrieveModelMixin,
                           mixins.ListModelMixin,
                           mixins.DestroyModelMixin):
+    """
+    Provides CUD methods that dispatch tasks with reservation that lock all distributions
```

Comment:
It also provides retrieve and list.

Reply:
Good catch, was getting too myopic in adding docstrings and there were none for this before now.